src/modules/commander/Commander.cpp

```diff
@@ -516,7 +523,7 @@ void usage(const char *reason)
 		PX4_INFO("%s", reason);
 	}
 
-	PX4_INFO("usage: commander {start|stop|status|calibrate|check|arm|disarm|takeoff|land|transition|mode}\n");
+	PX4_INFO("usage: commander {start|stop|status|calibrate|check|arm [-f]|disarm|takeoff|land|transition|mode}\n");
```

Comment:
Is there no cli docs for commander where this needs to be added as well?

Reply:
No, but I can add them.

Reply:
Done: https://github.com/PX4/Firmware/pull/12715/commits/802ba74b8a4ae4ba28f2747826366f486a54740c

Reply:
Nice!